src/sentry/workflow_engine/processors/detector.py

```diff
@@ -19,7 +18,7 @@ def get_detector_by_event(event_data: WorkflowEventData) -> Detector:
 
     if issue_occurrence is None:
         # TODO - @saponifi3d - check to see if there's a way to confirm these are for the error detector
-        detector = Detector.objects.get(project_id=evt.project_id, type=ErrorGroupType.slug)
+        detector = Detector.objects.get(project_id=evt.project_id, type=evt.group.issue_type.slug)
```

Comment:
i was running into a circular import error after importing `DetectorHandler` into the `grouping/grouptype.py` file, so i substituted `ErrorGroupType` with this